components/d2l-sequences-content-image.js

```
@@ -0,0 +1,56 @@
+import '../mixins/d2l-sequences-automatic-completion-tracking-mixin.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+export class D2LSequencesContentImage extends D2L.Polymer.Mixins.Sequences.AutomaticCompletionTrackingMixin() {
+	static get template() {
+		return html`
+		<style>
+		    img {
+                max-width: 100%;
+                height: auto;
+			}
+		</style>
+		<div>
+            <img src="[[_fileLocation]]">
+		</div>
+`;
+	}
+
+	static get is() {
+		return 'd2l-sequences-content-image';
+	}
+	static get properties() {
+		return {
+			href: {
+				type: String,
+				reflectToAttribute: true,
+				notify: true,
+				observer: '_scrollToTop'
+			},
+			_fileLocation: {
+				type: String,
+				computed: '_getFileLocation(entity)'
+			},
+			title: {
+				type: String,
+				computed: '_getTitle(entity)'
+			},
+		};
+	}
+	_scrollToTop() {
+		window.top.scrollTo(0, 0);
+	}
+	_getFileLocation(entity) {
+		try {
+			const fileActivity = entity.getSubEntityByClass('file-activity');
+			const file = fileActivity.getSubEntityByClass('file');
+			const link = file.getLinkByClass('pdf') || file.getLinkByClass('embed') || file.getLinkByRel('alternate');
+			return link.href;
+		} catch (e) {
+			return '';
+		}

```

Probably shouldn't have an empty catch statement. Was debugging and looks like this function is hit twice, once when entity is null and another time when the entity is populated so it could probably just be written like this instead: 

```
if (entity) {
   const fileActivity = entity.getSubEntityByClass('file-activity');
   const file = fileActivity.getSubEntityByClass('file');
   const link = file.getLinkByClass('embed') || file.getLinkByRel('alternate');
   return link.href;
} else {
  return '';
}
```
Note that we don't need `file.getLinkByClass('pdf')` either since we know it won't be a pdf file.

---

Will do. (Again just saw all the other files had it so added it for consistency: d2l-sequences-content{video, audio, file-pdf, etc.}.js)